[Catalog] Add all tests to both schemes.

romoore opened the pull request:
Adding all unit test and snapshot test targets to both Catalog and
Dragons schemes. It seems that at one point they both contained the same
targets, but were not updated consistently over time.

Partially implements #6935

Commit:
[Catalog] Add all tests to both schemes.

Adding all unit test and snapshot test targets to both Catalog and
Dragons schemes. It seems that at one point they both contained the same
targets, but were not updated consistently over time.

Closes #6935

yarneo reviewed (commented):
Testing failed:
	"_OBJC_CLASS_$_MDCActionSheetTestHelper", referenced from:
	Linker command failed with exit code 1 (use -v to see invocation)
** TEST FAILED **

Commit:
Merge branch 'develop' into issue-6935

romoore commented:
This PR should exclude that test target for now.  Once https://github.com/material-components/material-components-ios/issues/6957 is fixed, the tracking issue for this work can be picked up again.

Commit:
Remove ActionSheet Theming from schemes.

`MDCActionSheetThemingTest` depends on importing code from other
examples. However, that code is in a separate target (theming is Beta).
As a result, the extension fails to compile because the symbol
`MDCActionSheetTestHelper` can't be resolved.  Since it blocks the rest
of the tests being added to the schemes, it's being removed for now.

Commit:
Merge branch 'develop' into issue-6935
